--- origsrc/seahorse-2.22.3/configure.in	2008-05-26 13:13:10.000000000 -0500
+++ src/seahorse-2.22.3/configure.in	2008-07-09 22:32:06.015625000 -0500
@@ -27,6 +27,8 @@
 AC_CONFIG_MACRO_DIR([m4])
 
 AC_PROG_INTLTOOL([0.35.0])
+AC_PROG_CC
+AC_PROG_CXX
 AM_PROG_LIBTOOL
 AM_GCONF_SOURCE_2
 GNOME_DOC_INIT
--- origsrc/seahorse-2.22.3/libcryptui/Makefile.am	2007-11-24 15:38:27.000000000 -0600
+++ src/seahorse-2.22.3/libcryptui/Makefile.am	2008-07-09 22:32:06.031250000 -0500
@@ -37,7 +37,7 @@
     $(BUILT_SOURCES)
     
 libcryptui_la_LDFLAGS = \
-    -version-info $(LIBCRYPTUI_LT_RELEASE)
+    -version-info $(LIBCRYPTUI_LT_RELEASE) -no-undefined
     
 libcryptui_la_LIBADD = $(LIBCRYPTUI_LIBS) 
 
--- origsrc/seahorse-2.22.3/plugins/epiphany/Makefile.am	2008-05-26 12:34:59.000000000 -0500
+++ src/seahorse-2.22.3/plugins/epiphany/Makefile.am	2008-07-09 22:32:06.093750000 -0500
@@ -12,7 +12,7 @@
 
 libseahorseextension_la_CPPFLAGS = \
     -I$(top_srcdir)/include					\
-	-I$/mozilla	\
+	-I$(srcdir)/mozilla	\
 	-DSHARE_DIR=\"$(pkgdatadir)\"				\
     -DEPHY_EXTENSIONS_LOCALEDIR=\"$(datadir)/locale\"	\
 	$(AM_CPPFLAGS) \
